cmd/agola/cmd/maintenance.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2022 Sorint.lab
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+
+	"agola.io/agola/internal/errors"
+	gatewayclient "agola.io/agola/services/gateway/client"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+var cmdMaintenance = &cobra.Command{
+	Use: "maintenance",
+	Run: func(cmd *cobra.Command, args []string) {
+		if err := maintenance(cmd, args); err != nil {
+			log.Fatal().Err(err).Send()
+		}
+	},
+	Short: "maintenance",
+}
+
+type maintenanceOptions struct {
+	disable     bool
+	servicename string
+}
+
+var maintenanceOpts maintenanceOptions
+
+func init() {
+	flags := cmdMaintenance.Flags()
+
+	flags.BoolVarP(&maintenanceOpts.disable, "disable", "", true, "disable maintenance mode")
```

Comment:
Instead of an option that defaults to true it should be better to have two subcommands (enable and disable)